regression/contracts-dfcc/invar_havoc_dynamic_multi-dim_array_all_const_idx/debug

```diff
@@ -0,0 +1,2715 @@
+CBMC version 5.81.0 (cbmc-5.81.0-38-g8a964f611d-dirty) 64-bit x86_64 linux
```

Comment:
What is this file about? Is this perhaps an unintentional add?

Reply:
removed